continuous_integration/environment-3.8.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - tiledb-py
   - fsspec>=0.6.0
   - sqlalchemy
-  - pyarrow>=0.14.0,!=2.0.0
+  - pyarrow>=0.14.0
```

Comment:
Here just leave the "larger than" constraint, to get the last released version of pyarrow, which at the moment is 2.0.0 (after removing the `!=2.0.0` pin)